File: baek_joon/py_src/step9/1978.py
n = int(input())
a = list(map(int, input().split()))

count = 0

for i in a:
    b = 0
    for j in range(2, i):
        if i%j == 0:
            b = 1
    if i == 1:
        continue        
    if b == 0:
        count += 1    

print(count)

File: baekjoon/js/silverV/1978.js
// 소수 찾기
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');
const nums = input[1].split(' ').map(v => +v);
const isPrime = (num) => {
  if (num !== 1) {
    for (let i = 2; i <= Math.sqrt(num); i++) {
      if (num % i === 0) {
        return false;
      }
    }
    
    return true;
  } else {
    return false;
  }
};

console.log(nums.filter(num => isPrime(num)).length);